pages/implementation-status.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: default
+---
+
+Val's implementation is at a very early stage.
+Key components of the compiler are still to be implemented before a first version of the language can be used.
+Further, some aspects of the language design are still immature and will likely require more iterations.
+
+This page gives an overview of our roadmap going forward and provides details on our main milestones.
+
+![2022 roadmap](./implementation-status.png)
```

Comment:
This diagram doesn't look quite right to me.  Two-way interop is not in the picture, and it gives the impression that the specification is not being developed in parallel with the rest of the language.  Given that most projects never get around to a spec, that might look like a promise we're unlikely to fulfill.

Reply:
If there was a caption saying this is the **2022 Roadmap** that would help a lot.